src/SyntaxHighlighter/include/SyntaxHighlighter/X86Assembly.h

```diff
@@ -30,6 +30,9 @@ class X86Assembly : public QSyntaxHighlighter {
 
 void HighlightBlockAssembly(const QString& code,
                             std::function<void(int, int, const QTextCharFormat&)> set_format);
+void HighlightAnnotatingBlock(const QString& code,
+                              std::function<void(int, int, const QTextCharFormat&)> set_format,
+                              const QColor& default_color = Qt::white);
```

Comment:
Sorry I missed this in the last commit already: We shouldn't copy the `std::function` object. A const-ref should be more performant.

Reply:
ty